pilot/pkg/config/kube/crd/controller.go

```diff
@@ -58,13 +60,18 @@ var (
 		Help: "Errors converting k8s CRDs",
 	}, []string{"name"})
 
+	k8sResourceVersions = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "pilot_k8s_resource_versions",
+		Help: "Applied CRD resource versions",
+	}, []string{"gvk", "name", "namespace", "resourceVersion", "instanceID"})
```

Comment:
gvk ?

Reply:
GroupVersionKind, i.e. the fully qualified type of a k8s resource (see https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#types-kinds)

Reply:
maybe separate out to `apigroup` and `kind` ? that wouldn't really hurt in terms of cardinality of the metrics.